src/engraving/libmscore/segment.cpp

```diff
@@ -1655,8 +1655,7 @@ Spanner* Segment::firstSpanner(int activeStaff)
             if (s->startSegment() == this) {
                 if (e->staffIdx() == activeStaff) {
                     return s;
-                }
-                else if (e->isMeasure() && activeStaff == 0) {
+                } else if (e->isMeasure() && activeStaff == 0) {
                     return s;
```

Comment:
Could probably also be 
```cpp
    if (e->staffIdx() == activeStaff 
        || (e->isMeasure() && activeStaff == 0)) {
        return s;
    }
```

(Same for line 1683)

Reply:
fixed